services/121-service/src/migration/1724079447374-add-userId-to-twilio-message-entity.ts

```diff
@@ -0,0 +1,49 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUserIdToTwilioMessageEntity1724079447374
+  implements MigrationInterface
+{
+  name = 'AddUserIdToTwilioMessageEntity1724079447374';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."twilio_message" ADD "userId" integer`,
+    );
+
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."twilio_message" ALTER COLUMN "userId" SET NOT NULL`,
```

Comment:
I think this is not going to work because at line 10, you already add the column which set the column `userId` as NULL and then you try to update the column as NOT NULL.

This lines 13-15 should be after update the `userId` column values, no?

Reply:
Done